PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
Well, you did get about 5% right. I put your name back in https://github.com/notepad-plus-plus/notepad-plus-plus/pull/3024.

I just hate to see people getting alienated from using software in their own language. I know about a dozen Estonians that use Notepad++ daily and none of them have it set to Estonian. I based my translation entirely on the English file, because it was impossible to think of the existing one as a translation and not random gibberish.

I do appreciate your intent and effort, though.